.github/PULL_REQUEST_TEMPLATE/devconsole_pr.md

```diff
@@ -0,0 +1,29 @@
+
+# [JIRA-ID](add JIRA link): User Story / Defect / Task Title
+<!--For e.g [ODC-100](https://jira.coreos.com/browse/ODC-100): Title of user story / defect / task -->
+
+## Root cause
+<!--Briefly describe the root cause & analysis of the problem-->
+
+## Solution Description
+<!--Describe your code changes in detail and explain the solution-->
+
+- [ ] Is U/X approved?
+<!-- If designer review required, tag @openshift/team-devconsole-ux and add screenshots/gifs  -->
+
+- [ ] Covered Unit test cases / e2e test cases? 
+<!-- Were unit tests or E2E test recorded for this change, or was only manual testing applicable. If yes, attach report else state reason for not adding -->
+
+- [ ] Does this PR need regression testing of any feature / work flow?
+<!-- If the changes have bigger impact on any feature / work flow, do mention scope for regression testing -->
+
+- [ ] Does it have any dependancies?
+<!-- Mention dependancies like PR, Defect, Prerequisite setup, application, operators, etc -->
+
+### Is this PR tested on all browsers?
```

Comment:
Not sure all browsers are required... but showcasing where you did test it will help with PR reviews against other browsers.

Reply:
Agreed. Changed section title as "Browser conformance". Now Developers shall mark only tested browsers in the checklist.